package.json

```diff
@@ -240,6 +246,18 @@
         "title": "Upload Schema to Schema Registry",
         "category": "Confluent: Schemas"
       },
+      {
```

Comment:
The new command to evolve the exact highest version schema item within a subject group.